### PR TITLE
KubernetesV1: changed chmod from 777 to 755

### PR DESCRIPTION
**Pull request by v-mohithgc**
**Task name**: KubernetesV1

**Description**: found an elevation of privilege vulnerability in the KubernetesV1 Azure DevOps task. Permissions are being set too high using chmod 777. Which is considered a bad practice.

**Documentation changes required:** (Y/N) <Please mark if documentation changes are required>

**Added unit tests:** (Y/N) <Please mark if unit tests were added or updated according changes>

**Attached related issue:** (Y/N) <Please add link to related issue here>

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/KubernetesV1/package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "@types/uuid": "^8.3.0",
     "agent-base": "^6.0.2",
     "azure-pipelines-task-lib": "^4.0.1-preview",
-    "azure-pipelines-tasks-azure-arm-rest": "3.223.4",
+    "azure-pipelines-tasks-azure-arm-rest": "^3.226.0",
```
Comment (tkasparek) [inline Tasks/KubernetesV1/package.json:11]: Why is the version bump of the library needed?

Comment (v-mohithgc) [reply Tasks/KubernetesV1/package.json:11]: Hi @tkasparek , as a good practice while creating a PR, most of the time I execute 'npm audit fix' (which resolves CG vulnerabilities without breaking any changes), here the npm audit bumped this library version. I believe these library version change won't cause any issues and is certainly not the reason for the below CI check failure.

**Review by rvairavelu — APPROVED** [review]

**Review by manolerazvan — APPROVED** [review]

**Pull request merged**